websites/P/Peario/presence.ts

```diff
@@ -0,0 +1,115 @@
+const presence = new Presence({
+		clientId: "969204609845428234"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let cached: JSON;
```

Comment:
you should create an interface for the return type of these functions and use it on this variable and when returning below